### PR TITLE
metrics: fix the table compatibility problem

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Issue Number: Ref #5396.

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Manual test

### Release note

```release-note
None
```

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

**Comment by nolouch** [comment]
how the grafana looks like?

File: metrics/grafana/pd.json [hunk 1]
```diff
@@ -1231,7 +1196,7 @@
             "h": 7,
             "w": 4,
             "x": 0,
-            "y": 14
+            "y": 20
```
Comment (bufferflies) [inline metrics/grafana/pd.json:1199]: why change the position? I can not see any new monitor .

**Review by bufferflies — APPROVED** [review]
rest LGTM

**Review by disksing — APPROVED** [review]

**Pull request merged**